lib/developer_portal/lib/liquid/template/fallback_resolver.rb

```diff
@@ -8,19 +8,8 @@ def initialize(path = DeveloperPortal::VIEW_PATH)
       end
```

Comment:
can we also remove this constructor? Or the default value is important? Why is default value not important with `FallbackResolverNoPrefix`?

Reply:
I think provided `path` is needed in the constructor. In `FallbackResolverNoPrefix` not that it's not important, but it's just that it inherits from `FallbackResolver`.

Reply:
The `FileSystemResolver` has
```
def initialize(path)
```

So the `path` argument exists. The difference is that we add a default value. So the question is whether we need this default value here as well in `FallbackResolverNoPrefix`.

Reply:
Yes, what I mean is that as `FallbackResolverNoPrefix` is a subclass of `FallbackResolver`, it inherits that constructor, and it already has that default value for `path`.